google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/HttpStorageRpc.java

```diff
@@ -226,6 +226,7 @@ public void onFailure(GoogleJsonError googleJsonError, HttpHeaders httpHeaders)
   }
 
   private static StorageException translate(IOException exception) {
+
```

Comment:
Do we run a formatter for java? Is this blank line a problem for it?

Reply:
It doesn't clean it up surprisingly.